File: app/src/main/java/com/paycraft/ticketbooker/models/ConfirmBooking.java
package com.paycraft.ticketbooker.models;

public class ConfirmBooking {

    private boolean success;
    private String ticketId;

    public ConfirmBooking(boolean success, String ticketId) {
        this.success = success;
        this.ticketId = ticketId;
    }

    public boolean isSuccess() {
        return success;
    }

    public void setSuccess(boolean success) {
        this.success = success;
    }

    public String getTicketId() {
        return ticketId;
    }

    public void setTicketId(String ticketId) {
        this.ticketId = ticketId;
    }

    @Override
    public String toString() {
        return "ConfirmBooking{" +
                "success=" + success +
                ", ticketId='" + ticketId + '\'' +
                '}';
    }
}

File: app/src/main/java/com/paycraft/ticketbooker/models/ConfirmBooking.kt
package com.paycraft.ticketbooker.models

import com.google.gson.annotations.SerializedName

class ConfirmBooking(
        @field:SerializedName("success")
        var isSuccess: Boolean, var ticketId: String?) {

    override fun toString(): String {
        return "ConfirmBooking{" +
                "success=" + isSuccess +
                ", ticketId='" + ticketId + '\''.toString() +
                '}'.toString()
    }
}

